protostar/commands/test/testing_live_logger.py

```diff
@@ -124,16 +121,22 @@ def log_human(
                             progress_bar.update(1)
                             tests_left_n -= 1
                 finally:
-                    progress_bar.write("")
-                    progress_bar.clear()
+                    progress_bar.close()
```

Comment:
So `progress_bar.close()` == `progress_bar.write("")` + `progress_bar.clear()`? Or is the behavior different?
Asking out of curiosity.

Reply:
This way we guarantee that the progress bar actually disappears without filling completely, that wasn't the case before.